iree/test/micro_benchmarks/mhlo_dot.mlir

```diff
@@ -0,0 +1,41 @@
+func @dot_384x384x512() -> tensor<384x512xf32> attributes { iree.module.export } {
```

Comment:
Same as the conv ops, can we document where we are getting these from.

Reply:
BTW, These ops also don't have to be sampled from a particular model (this coincidentally happened now as we are tracking e2e models performance and these ops are the slow path) they are mainly added to exercise  the codegen perf, e.g we can add an 3x4 conv to benchmark a winograd vs direct-conv implementation .